docs/get-started-cloud.md

```diff
@@ -9,176 +9,252 @@ weight=2
-memory and CPUs (by default `docker-machine` creates a small VPS). For a full
-list of the flags/settings available and their defaults, see the output of
-`docker-machine create -h`.
-
-When the creation of a host is initiated, a unique SSH key for accessing the
-host (initially for provisioning, then directly later if the user runs the
-`docker-machine ssh` command) will be created automatically and stored in the
-client's directory in `~/.docker/machines`. After the creation of the SSH key,
-Docker will be installed on the remote machine and the daemon will be configured
-to accept remote connections over TCP using TLS for authentication. Once this
-is finished, the host is ready for connection.
-
-To prepare the Docker client to send commands to the remote server we have
-created, we can use the subshell method again:
-
-    $ eval "$(docker-machine env staging)"
-
-From this point, the remote host behaves much like the local host we created in the last section. If we look at `docker-machine ls`, we'll see it is now the
-"active" host, indicated by an asterisk (`*`) in that column:
+  1. Go to the Digital Ocean administrator console and click **API** in the header.
 
-    $ docker-machine ls
-    NAME      ACTIVE   DRIVER         STATE     URL
-    dev       -        virtualbox     Running   tcp://192.168.99.103:2376
-    staging   *        digitalocean   Running   tcp://203.0.113.81:2376
+    ![Click API in Digital Ocean console](img/ocean_click_api.png)
 
-To remove a host and all of its containers and images, use `docker-machine rm`:
+  2. Click **Generate New Token** to get to the token generator.
 
-    $ docker-machine rm dev staging
-    Do you really want to remove "dev"? (y/n): y
-    Successfully removed dev
-    Do you really want to remove "staging"? (y/n): y
-    Successfully removed staging
+    ![Generate token](img/ocean_gen_token.png)
 
-    $ docker-machine ls
-    NAME      ACTIVE   DRIVER       STATE     URL
+  3. Give the token a clever name (e.g. "machine"), make sure the **Write (Optional)** checkbox is checked, and click **Generate Token**.
 
-### Docker supported drivers
+    ![Name and generate token](img/ocean_token_create.png)
 
-Docker Machine drivers are available for several other cloud providers. For a full list, see [Supported Drivers](https://docs.docker.com/machine/drivers/).
+  4. Grab (copy to clipboard) the generated big long hex string and store it somewhere safe.
 
-### Docker Machine command and driver reference
+    ![Copy and save personal access token](img/ocean_save_token.png)
 
--   `docker-machine` [create](https://docs.docker.com/machine/reference/create/) command
--   [Driver options and operating system defaults](https://docs.docker.com/machine/drivers/os-base/)
+    This is the personal access token you'll use in the next step to create your cloud server.
 
-### 3rd-party driver plugins
+### Step 3. Start Docker Machine
```

Comment:
@dmp42 , good point .. this section needs revision. I do want to have them run some `docker` commands to make sure that both `docker-machine` and `docker` are working, but the examples in Step 3 are confusing. The only other thing I'd mention here, is that if you installed with Toolbox, you will have a default machine, so I might still be tempted to show it here. But, yes, this section needs revision ... coming up.